contracts/StandardAssetRegistry.sol

```diff
@@ -154,16 +154,16 @@ contract StandardAssetRegistry is AssetRegistryStorage, IAssetRegistry {
   //
 
   function generate(uint256 assetId) public {
-    generate(assetId, msg.sender, '');
+    doGenerate(assetId, msg.sender, '');
   }
 
   function generate(uint256 assetId, string data) public {
-    generate(assetId, msg.sender, data);
+    doGenerate(assetId, msg.sender, data);
   }
 
-  function generate(uint256 assetId, address _beneficiary, string data) public {
-    doGenerate(assetId, _beneficiary, data);
-  }
+//  function generate(uint256 assetId, address _beneficiary, string data) public {
```

Comment:
Never commit commented code. Why are you commenting this out?

Reply:
Sorry, it was on purpose, @abarmat told me to submit the code as it is so he can try why overload functions are not working as expected. 

Not going to happen again.